### ISSUE TITLE
Document how to use hotfix_source_repo

### ISSUE THREAD

**Issue by dhirschfeld** [conda/conda-build#3513]
IIUC the raw output from `conda index` is no longer useful as conda won't be able install packages based on it - instead, the raw output must be patched with the *hotfixes* in a given repo.

It appears that for anaconda packages the hotfixes live at:
 - https://github.com/AnacondaRecipes/repodata-hotfixes

...whereas for conda-forge the hotfixes live at:
 - https://github.com/conda-forge/conda-forge-repodata-patches-feedstock

Am I correct so far?

I was then trying to grok how to apply these hotfixes when I came accross the `hotfix_source_repo` argument in the `update_index` function:
https://github.com/conda/conda-build/blob/12319d357f743ab21d2bf954b889e3a3f0e5b3af/conda_build/index.py#L296-L299

It seems like this might do exactly what I'm after but I'm not too sure how to use it.

**Comment by dhirschfeld** [comment]
I'm imagining a syntax such as:
```
conda index --hotfix-source-repo=https://github.com/AnacondaRecipes/repodata-hotfixes
```
...but maybe that's not correct and it needs a local checkout?

It would be good if someone could let me know if I'm on the right track.

**Comment by dhirschfeld** [comment]
Just waiting on the [cloudflare CDN issue](https://gitter.im/conda-forge/conda-forge.github.io?at=5ce1e78b63dea422b4b022b7) before I can install the latest and actually test it out...

**Comment by dhirschfeld** [comment]
Confirmation from gitter:
https://gitter.im/conda-forge/conda-forge.github.io?at=5dcb1943fd6fe41fc07209e3

> *The hotfixing logic is part of `conda index` which is included in `conda-build`. The fixes are passed via the `-p` or `--hotfix-source-repo` arguments.*

**Comment by dhirschfeld** [comment]
Closed by #3931
